### DISCUSSION TITLE
Force formulas from specific tap to be have higher priority

### DISCUSSION THREAD

**Discussion by sycured** [Homebrew/discussions#1674] [Everyday usage]
Hi, I use some taps and want exclusively packages from those taps instead of the "base" homebrew.
Do you have a solution like apt policy where each repository can have a custom priority and force update from that repository.
Thanks

**Comment by SMillerDev** [comment]
No, core formula will always have the highest priority.

**Reply by sycured** [reply]
For example, I have hashicorp's tap and I would like to have it with the highest priority, the same about curl from cloudflare's tap…
Have you a possible workaround?

**Reply by tobylane** [reply]
Give the full name, cloudflare/cloudflare/curl.

**Reply by sycured** [reply]
Ok thank you
